### PR TITLE
Small global_lists.dm optimizations

### PR DESCRIPTION
**Pull request by TheShown911**
## About The Pull Request

Rewrites some lists in global_lists.dm with global list macros and removes a few unused global lists.

## Why It's Good For The Game

This should lower memory consumption by a bit and make code look nicer.

## Changelog
:cl: TheShown
code: Updated global_lists.dm with global list macros, removed a few unused global lists.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_hud.dm [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
+//	var/datum/hud/human/HUDdatum = GLOB.HUDdatums[H.defaultHUD]
 	var/recreate_flag = FALSE
 
 	if(!check_HUDdatum())//check client prefs
 		log_debug("[H] try check a HUD, but HUDdatums not have \"[H.client.prefs.UI_style]!\"")
 		to_chat(H, "Some problem hase accure, use default HUD type")
 		H.defaultHUD = "ErisStyle"
 		recreate_flag = TRUE
-	else if (H.client.prefs.UI_style != H.defaultHUD)//���� ����� � ���� �� ��������� �� ������ � �������
+	else if (H.client.prefs.UI_style != H.defaultHUD)//���� ����� � ���� �� ��������� �� ������ � �������
```
Comment (TheShown911) [inline code/modules/mob/living/carbon/human/human_hud.dm:14]: Not sure what happened here?

Comment (drexample) [reply code/modules/mob/living/carbon/human/human_hud.dm:14]: fix eeeet

Comment (TheShown911) [reply code/modules/mob/living/carbon/human/human_hud.dm:14]: Can i just delete all the comments, mr.headdev?

Comment (mattroks101) [reply code/modules/mob/living/carbon/human/human_hud.dm:14]: I would yeah. Or replace them with an English counterpart.

Comment (TheShown911) [reply code/modules/mob/living/carbon/human/human_hud.dm]: Could not find English counterparts for "Âëèâàåì â ñîîòâåñòâóþùèé ñïèñîê", "Èùåì òàêîé â äàòóìå", "åñëè ó êëèåíòà ìîáà ïðîïèñàí ñòèëü\òèï ÕÓÄà" and other comments so they have been deleted.

File: code/modules/mob/living/silicon/robot/robot_hud.dm [hunk 2]
```diff
@@ -25,14 +25,14 @@
 /mob/living/silicon/robot/check_HUDdatum()//correct a datum?
 	var/mob/living/silicon/robot/H = src
 
-	if (H.defaultHUD == "BorgStyle") //���� � ������� ���� �������� �����\��� ����
-		if(global.HUDdatums.Find(H.defaultHUD))//���� ���������� ����� ��� ����
+	if (H.defaultHUD == "BorgStyle") //���� � ������� ���� �������� �����\��� ����
```
Comment (mattroks101) [inline code/modules/mob/living/silicon/robot/robot_hud.dm:28]: Oh god it happened again. @drexample do these need to be removed as well?

Comment (TheShown911) [reply code/modules/mob/living/silicon/robot/robot_hud.dm:28]: This is actually not corrupted unciode but some strange version of russian.

Comment (drexample) [reply code/modules/mob/living/silicon/robot/robot_hud.dm:28]: DESTROY THE CHIL- 
As long as it compiles, probably some github-byond-git unicode bullshit.
Yay for optimizations.

**Review by drexample — APPROVED** [review]

**Pull request merged**